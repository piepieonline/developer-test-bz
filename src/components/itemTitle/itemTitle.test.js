import React from 'react';
import ReactDOM from 'react-dom';
import ItemTitle from './itemTitle';

it('renders the expected HTML', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ItemTitle title="title" subtitle="subtitle" />, div);

    expect(div).toMatchSnapshot();

    ReactDOM.unmountComponentAtNode(div);
});