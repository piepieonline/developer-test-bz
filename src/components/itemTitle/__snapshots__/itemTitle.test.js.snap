// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the expected HTML 1`] = `
<div>
  <header
    class="item-title"
  >
    <h1
      class="item-title-heading"
    >
      title
    </h1>
    <span
      class="item-title-subtitle"
    >
      subtitle
    </span>
  </header>
</div>
`;
